Missing dependency 'read-installed'

After a fresh install via npm I get the following error

```
➜  dat  dat init

module.js:340
    throw err;
    ^
Error: Cannot find module 'read-installed'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/dat/lib/backend.js:2:21)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

--- a/package.json
+++ b/package.json
@@ -60,7 +60,8 @@
     "multilevel": "~5.5.0",
     "levelup": "~0.18.2",
     "leveldown": "~0.10.2",
-    "csv-write-stream": "~0.1.1"
+    "csv-write-stream": "~0.1.1",
+    "read-installed": "~0.2.5"
   },
   "directories": {
     "example": "examples",

